migrations/0002_06.08.2021-migrations.down.sql

```diff
@@ -0,0 +1,16 @@
+-- +migrate Down
+DROP TABLE IF EXISTS users cascade;
```

Comment:
when deleting all db, cascade will only make it longer, the result will be same

Reply:
don't think we will  feel any performance issues here